drivers/kw41zrf/kw41zrf.c

```diff
@@ -43,9 +44,11 @@ static void kw41zrf_set_address(kw41zrf_t *dev)
     /* get unique IDs to use as hardware addresses */
     luid_get_eui64(&addr_long);
     luid_get_short(&addr_short);
+    /* this places the unique bits in less significant places */
+    addr_long.uint64.u64 = byteorder_swapll(addr_long.uint64.u64);
```

Comment:
Huh? `eui64_t` should always be in network byte order.
The whole point of having that type is that the byte order is well-defined.

Reply:
That's why the comment clarifies that the swap is just to get a better uid to make an address from. This is mainly to avoid functional changes from previous behavior but also it's really terrible having the ipv6 addresses of many nodes all end the same.